src/page-response-impact/PopulationImpactMetrics.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import ambulanceIcon from "./icons/ic_ambulance.svg";
+import heartIcon from "./icons/ic_heart.svg";
+import ImpactMetricCard from "./ImpactMetricCard";
+import ImpactMetricsContainer from "./ImpactMetricsContainer";
+
+const TitleSpan = styled.span`
+  color: ${Colors.teal};
+`;
+
+const Title = ({ title }: { [title: string]: string }) => {
+  return (
+    <>
+      <span>{title}</span> <TitleSpan>reduced by</TitleSpan>
```

Comment:
probably not a single line fix, but mocks are `font-weight: normal` for these titles.

Reply:
good catch!